Kuroba/app/src/main/res/values/strings.xml

```diff
@@ -514,32 +514,32 @@ Enabled filters have priority from top to bottom in your list. Filter precedence
     <string name="setting_image_auto_load_wifi">Wi-Fi only</string>
     <string name="setting_image_auto_load_none">Never</string>
 
-    <string name="setting_video_auto_load">Automatically load videos</string>
-    <string name="setting_video_auto_loop">Enable automatic video-looping</string>
-    <string name="setting_video_auto_loop_description">Automatically loop video content</string>
-    <string name="setting_auto_load_thread_images">Enable media pre-fetching</string>
-    <string name="setting_auto_load_thread_images_description">Instead of loading thumbnails, pre-loads full size images into the file cache when thumbnails should be displayed and uses them for thumbnail display. Tapping a thumbnail will load the full size image directly from the cache.</string>
+    <string name="setting_video_auto_load">Auto-load videos</string>
+    <string name="setting_video_auto_loop">Automatic video-looping</string>
+    <string name="setting_video_auto_loop_description">Start videos from the beginning once finished</string>
+    <string name="setting_auto_load_thread_images">Media pre-fetching</string>
+    <string name="setting_auto_load_thread_images_description">Pre-loads fullsize images to show as thumbnails. Tapping thumbnails loads fullsize the pre-fetched image directly.</string>
     <string name="save_location_screen">Save location</string>
```

Comment:
This is a bit nondescript. Verb or noun?

Reply:
It however likely fails in translation, where it possibly becomes an active error.